common/firmware/STM32L562QEIx/STM32L562QEIx_FLASH.ld

```diff
@@ -37,33 +33,34 @@
 ENTRY(Reset_Handler)
 
 /* Highest address of the user mode stack */
-_estack = ORIGIN(RAM) + LENGTH(RAM);	/* end of "RAM" Ram type memory */
-
-_Min_Heap_Size = 0x400 ;	/* required amount of heap  */
-_Min_Stack_Size = 0x800 ;	/* required amount of stack */
+_estack = 0x20030000;    /* end of RAM */
+/* Generate a link error if heap and stack don't fit into RAM */
+_Min_Heap_Size = 0x200;      /* required amount of heap  */
+_Min_Stack_Size = 0x400; /* required amount of stack */
 
-/* Memories definition */
+/* Specify the memory areas */
 MEMORY
 {
-  RAM	(xrw)	: ORIGIN = 0x20000000,	LENGTH = 192K
-  ROM	(rx)	: ORIGIN = 0x8000000,	LENGTH = 512K
+FLASH (rx)     : ORIGIN = 0x08000000, LENGTH = 512K
+RAM (xrw)      : ORIGIN = 0x20000000, LENGTH = 256K
+OSPI (rx)      : ORIGIN = 0x90000000, LENGTH = 16M
 }
 
-/* Sections */
+/* Define output sections */
 SECTIONS
 {
-  /* The startup code into "ROM" Rom type memory */
+  /* The startup code goes first into FLASH */
   .isr_vector :
   {
-    . = ALIGN(8);
+    . = ALIGN(4);
     KEEP(*(.isr_vector)) /* Startup code */
-    . = ALIGN(8);
-  } >ROM
+    . = ALIGN(4);
+  } >FLASH
 
-  /* The program code and other data into "ROM" Rom type memory */
+  /* The program code and other data goes into FLASH */
   .text :
   {
-    . = ALIGN(8);
+    . = ALIGN(4);
```

Comment:
actually do you know why this characteristically aligns on 4-byte boundaries instead of 8?